lib/oli_web/live/admin/vr_user_agents_view.ex

```diff
@@ -0,0 +1,201 @@
+  defp columns_title_data, do: [{"id", "ID"}, {"user_agent", "User agent"}, {"nil", "Delete"}]
+
+  def mount(_, _session, socket) do
+    data_manager = %{sort: @initial_sort, paginate: @initial_paginate}
+
+    vr_user_agents =
+      VrUserAgents.vr_user_agents(sort: data_manager.sort, paginate: data_manager.paginate)
+
+    form = VrUserAgent.new_changeset() |> to_form()
+
+    socket =
+      socket
+      |> assign(breadcrumbs: breadcrumb())
+      |> assign(vr_user_agents: vr_user_agents)
+      |> assign(form: form)
+      |> assign(data_manager: data_manager)
+      |> assign(total_count: VrUserAgents.count())
+
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~H"""
+    <div>
+      <.form for={@form} phx-submit="add_user_agent" phx-change="change_content">
+        <div class="flex flex-col gap-2 w-full">
+          <.input
+            id="entered_text"
+            type="textarea"
+            field={@form[:user_agent]}
+            placeholder="Enter user agent to add"
+            autofocus
+            class="w-full focus:ring-2 focus:ring-blue-300 rounded-lg text-sm px-5 py-1.5 dark:bg-gray-800"
+          />
+          <button
+            type="submit"
+            class="w-1/12 px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:outline-none dark:bg-blue-600 dark:hover:bg-blue-700"
+          >
+            Add
+          </button>
+        </div>
+      </.form>
+      <Paging.render
+        id="header_paging"
+        total_count={@total_count}
+        offset={@data_manager.paginate.offset}
+        limit={@data_manager.paginate.limit}
+        click={JS.push("paged_table_page_change")}
+      />
+      <table class="min-w-full border ">
+        <thead>
+          <tr>
+            <th
+              :for={{sort_field, col_title} <- columns_title_data()}
+              phx-click="sort_by"
+              phx-value-sort-column={sort_field}
+            >
+              <%= col_title %>
+              <% sort_data = @data_manager[:sort] %>
+              <i :if={match?(^sort_data, {:asc, sort_field})} class="fa fa-sort-up"></i>
+              <i :if={match?(^sort_data, {:desc, sort_field})} class="fa fa-sort-down"></i>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr :for={vr_user_agent <- @vr_user_agents}>
+            <td class="mx-auto text-center">
+              <%= vr_user_agent.id %>
+            </td>
+            <td class="mx-auto text-center">
+              <%= vr_user_agent.user_agent %>
+            </td>
+            <td class="mx-auto text-center">
+              <button
+                type="button"
+                phx-click="delete_vr_entry"
+                phx-value-id={vr_user_agent.id}
+                class="focus:outline-none text-white bg-red-700 hover:bg-red-800 focus:ring-4 focus:ring-red-300 font-medium rounded-lg text-sm px-5 py-1.5 me-2 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-900"
```

Comment:
This is a non-standard button, relative to the styles that we use in all of the other views.  Same thing with the "Add" button.   Please get button styles from another view so that these are consistent.